lib/crypto.js

```diff
@@ -647,9 +647,21 @@ exports.setEngine = function setEngine(id, flags) {
   return binding.setEngine(id, flags);
 };
 
-exports.randomBytes = exports.pseudoRandomBytes = randomBytes;
-
-exports.rng = exports.prng = randomBytes;
+exports.randomBytes = util._deprecateSync(randomBytes,
```

Comment:
@silverwind  I tested it locally like this,

    ➜  io.js git:(master) ✗ cat test.js 
    const crypto = require('crypto');

    console.log(crypto.randomBytes(5));
    crypto.randomBytes(5, console.log);
    ➜  io.js git:(master) ✗ ./iojs test.js
    <Buffer 11 aa eb 88 d2>
    (node) randomBytes used as async is deprecated. Use randomBytesSync instead.
    null <Buffer 01 9f 27 82 fd>